src/cli/lib/config.ts

```diff
@@ -34,7 +35,7 @@ export function getBabelConfig(dir: string = process.cwd()): any {
  * @returns {object} The ReChunk configuration object.
```

Comment:
🟡 we can more exactly specify the returning type in JSDoc

and same for `getPackageJson`

```suggestion
 * @returns {ReChunkConfig} The ReChunk configuration object.
```

Reply:
great catch! updated!